teamlist.md

```diff
@@ -0,0 +1,64 @@
+Teams
+-----
+
+##### This is the team list as specified in cEP-0003
+
+- **Admin**:
+Admin team for the self hostet coala servers and ci.
+
+- **Bears**:
+Developing and maintaining the coala-bears repository. Wrap more
+existing tools, help with writing new bears and update existing ones.
+
+- **cib**:
+The cib team develops and maintains the cib tool for coala.
+
+- **Community**:
+The community team organizes community bonding events like meetups and
+gaming evenings to support and promote a healthy, happy community.
+It also serves as a contact for personal problems or conflicts between
+community members.
+
+- **Core**:
+Developing and maintaining the coala core repository.
+In addition everything that does not fit into another team's domain,
+will be assigned to the core team.
+
+- **Design**;
+Creating mockups and designs for things like a homepage, gui and logos
+for posters, sirts and flyers.
+
+- **Docs**:
+Improve the existing documentation, wiki, tutorials and guides.
+Introduce new ones as needed.
+
+- **Engagement**:
+Create all promotional material like flyers, blog posts,homepage
+content, social media presence and such. Also manage conference
+attendance and feedback handling.
+
+- **Gitlab**
+The Gitlab team leads the migration of the coala repositories to Gitlab.
+This team will disband after the migration and mirroring are all done.
```

Comment:
so far we don't have an application for this anyway so we could remove it and if someone is interested he/she has to apply again. Didn't @jayvdb kinda work on this?